src/dynamic_honey_badger/builder.rs

```diff
@@ -138,6 +150,7 @@ where
         let arc_netinfo = Arc::new(netinfo.clone());
         let honey_badger = HoneyBadger::builder(arc_netinfo.clone())
             .max_future_epochs(self.max_future_epochs)
+            .encryption_schedule(self.encryption_schedule)
```

Comment:
Actually, I'm wondering whether the encryption schedule needs to be part of the `JoinPlan`, since the joining node must use the same one as the others.

Reply:
Ah, I see. That's what I had intended, just got it wrong and didn't revisit it.